server/etcdserver/v3_server.go

```diff
@@ -367,6 +392,23 @@ func (s *EtcdServer) LeaseTimeToLive(ctx context.Context, r *pb.LeaseTimeToLiveR
 	return nil, errors.ErrCanceled
 }
 
+func (s *EtcdServer) LeaseTimeToLive(ctx context.Context, r *pb.LeaseTimeToLiveRequest) (*pb.LeaseTimeToLiveResponse, error) {
+	err, rev := s.checkLeaseTimeToLive(ctx, lease.LeaseID(r.ID))
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := s.leaseTimeToLive(ctx, r)
+	if err != nil {
+		return nil, err
+	}
+
+	if rev != s.AuthStore().Revision() {
```

Comment:
I am thinking probably we should just skip the check when auth isn't enabled. For example, when auth isn't enabled, users may also update RBAC to cause the autuRev to increase; obviously we shouldn't return `auth.ErrAuthOldRevision` in that case.
```suggestion
	if s.AuthStore().IsAuthEnabled() && rev != s.AuthStore().Revision() {
```

Reply:
Thanks, it makes sense! Let me update this PR.